src/jit/emitxarch.cpp

```diff
@@ -1494,16 +1494,15 @@ inline UNATIVE_OFFSET emitter::emitInsSizeRR(instruction ins, regNumber reg1, re
     emitAttr size = EA_SIZE(attr);
 
     UNATIVE_OFFSET sz;
-#ifdef _TARGET_AMD64_
-    // If Byte 4 (which is 0xFF00) is non-zero, that's where the RM encoding goes.
+
+    // If Byte 4 (which is 0xFF00) is zero, that's where the RM encoding goes.
```

Comment:
No, you're not confused - I am. It's unfortunate that the comment begins with the opposite of the condition being checked. So NVM (unless you'd like to change the comment to "If Byte 4 (which is 0xFF00) is non-zero, then it will be placed after the 4 byte encoding, making the total 5 bytes. Otherwise, that byte is where the RM encoding goes." Sorry for my confusion.

---
In reply to: [90501469](https://github.com/dotnet/coreclr/pull/8414#discussion_r90501469) [](ancestors = 90501469)